new-cdc/eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/KafkaOffsetStoreConfiguration.java

```diff
@@ -21,7 +21,9 @@ public OffsetStore offsetStore(EventuateKafkaConfigurationProperties eventuateKa
                                  EventuateKafkaConsumerConfigurationProperties eventuateKafkaConsumerConfigurationProperties) {
 
     return new DatabaseOffsetKafkaStore(eventuateConfigurationProperties.getOffsetStorageTopicName(),
-            eventuateConfigurationProperties.getReaderName(),
+            eventuateConfigurationProperties.getOffsetStoreKey() == null
```

Comment:
`foo.x()` and `foo.y()` and `foo.z()` should be combined into single call `foo.xyz()`

Reply:
Chris, I moved condition inside the eventuateConfigurationProperties.getOffsetStoreKey(), is it ok?